resources/incubator/images/spindle/network.svg

```diff
@@ -0,0 +1,613 @@
+         y="50.408062"
+         style="stroke-width:0.264583"
+         id="tspan38717">Querier</tspan></text>
+    <text
+       xml:space="preserve"
+       style="font-style:normal;font-weight:normal;font-size:10.5833px;line-height:1.25;font-family:sans-serif;fill:#000000;fill-opacity:1;stroke:none;stroke-width:0.264583"
+       x="63.071732"
+       y="160.29753"
+       id="text38723"><tspan
+         sodipodi:role="line"
+         id="tspan38721"
+         x="63.071732"
+         y="160.29753"
+         style="stroke-width:0.264583" /></text>
+    <text
+       xml:space="preserve"
+       style="font-style:normal;font-weight:normal;font-size:10.5833px;line-height:1.25;font-family:sans-serif;fill:#000000;fill-opacity:1;stroke:none;stroke-width:0.265;stroke-miterlimit:4;stroke-dasharray:none"
+       x="75.544762"
+       y="72.956696"
+       id="text38871"><tspan
+         sodipodi:role="line"
+         x="75.544762"
+         y="72.956696"
+         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:3.52778px;font-family:sans-serif;-inkscape-font-specification:'sans-serif, Normal';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-east-asian:normal;stroke-width:0.265;stroke-miterlimit:4;stroke-dasharray:none"
+         id="tspan38881">ask to train model</tspan></text>
+    <path
+       style="fill:none;stroke:#000000;stroke-width:0.264583px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1;marker-start:url(#marker38729-6)"
+       d="m 109.6131,61.799103 c 0,21.922627 0,21.922627 0,21.922627"
+       id="path38725-4" />
+    <text
+       xml:space="preserve"
+       style="font-style:normal;font-weight:normal;font-size:10.5833px;line-height:0.35;font-family:sans-serif;fill:#000000;fill-opacity:1;stroke:none;stroke-width:0.264999;stroke-miterlimit:4;stroke-dasharray:none"
+       x="111.56572"
+       y="70.506142"
+       id="text38871-1"><tspan
+         sodipodi:role="line"
+         x="111.56572"
+         y="70.506142"
+         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:3.52777px;font-family:sans-serif;-inkscape-font-specification:'sans-serif, Normal';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-east-asian:normal;stroke-width:0.264999;stroke-miterlimit:4;stroke-dasharray:none"
+         id="tspan38881-2">when model is trained,</tspan><tspan
+         sodipodi:role="line"
+         x="111.56572"
+         y="74.921173"
+         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:3.52777px;font-family:sans-serif;-inkscape-font-specification:'sans-serif, Normal';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-east-asian:normal;stroke-width:0.264999;stroke-miterlimit:4;stroke-dasharray:none"
+         id="tspan39884">get prediction</tspan></text>
```

Comment:
Suggestion: I would separate this in two parts:
1) The querier asks a node to train the model, which results in a globally trained model.
2) The querier sends data to a node and receives a prediction as the answer.

Something like:
```
--------------------------------------------------------------------------------------------------
|                                             Querier                                            |
--------------------------------------------------------------------------------------------------
                |  ↑                                                     |  ↑
    train model |  | done (model is trained)                       query |  | prediction
                ↓  |                                                     ↓  |
--------------------------------------------------------------------------------------------------
|                                            Hospital                                            |
--------------------------------------------------------------------------------------------------
```

Reply:
Good point, I tried to simplify it by merging the two calls, but that doesn't helps with the following demo. I've splitted it.